SKILL.tsv

```diff
@@ -4809,14 +4809,14 @@ SKILL_20150717_004808	$Increases the AoE Attack Ratio of [Claymore] by 1 per att
 SKILL_20150717_004809	As the enemy that was flown to the air by [Punji Stake] falls down on the ground, it will receive damage equal to 50% of Physical ATK and the enemies nearby will also receive damage.
 SKILL_20150717_004810	$Decreases the enemy's defense that are attacked with [Retrieve] by 10 per attribute level.
 SKILL_20150717_004811	{memo X}$Provides immunity to [Poison] for 2 seconds per attribute level to a character whose [Poison] has been cured due to [Detoxify].
-SKILL_20150717_004812	Even if an object gets out of the range of [Throw Gu Pot], [Poison] will last for 2 secs per Attribute level.
-SKILL_20150717_004813	The enemies that are in the range of [Scan] will fall into Confusion with a certain probability for 5 secs. The probability will increase by 10% per Attribute level.
+SKILL_20150717_004812	Even if an object gets out of the range of [Throw Gu Pot], [Poison] will continue for 2 seconds per attribute level.
+SKILL_20150717_004813	The enemies that are in the range of [Scan] will fall into [Confusion] with a certain probability for 5 seconds. The probability will increase by 10% per attribute level.
```

Comment:
Enemies within the range of [Scan] will be afflicted with [Confusion] with a certain chance for 5 seconds. Increases the chance by 10% per attribute level.